python/SNAPS_assigner.py

```diff
@@ -128,7 +129,13 @@ def read_config_file(self, filename):
             self.logger.error("Some required parameters were missing/not imported: %s",
                                 ", ".join(missing_pars))
 
-        self.logger.info("Finished reading in config parameters from %s"
+        par_dir = Path(filename).parent
```

Comment:
Could you elaborate on exactly what this is doing in a comment? I'm probably being a bit dim, but I can't see how this section works.